bin/sandbox

```diff
@@ -0,0 +1,8 @@
+# Hooks will not work correctly due to linked modules from sandbox unless the react version is linked:
```

Comment:
So basically, if you run:

```
npm run sandbox
```

Then later, run
```
npm run test
```

It will error with the following:

```
→npm run test

> @elastic/react-search-ui-views@1.3.2 test /Users/jasonstoltzfus/workspace/search-ui/packages/react-search-ui-views
> ./bin/test

ERROR: You have some node_modules linked. Please unlink any node_modules or your tests may not run correctly.
This may have happened as a result of running 'npm run sandbox' from the project root.
Unlink any linked dependencies using the command 'npm unlink <dependency_name> --no-save'

The modules you have linked are:
74543758 0 lrwxr-xr-x 1 jasonstoltzfus staff 65 Mar 26 15:47 node_modules/react -> ../../../../../.nvm/versions/node/v10.16.0/lib/node_modules/react

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @elastic/react-search-ui-views@1.3.2 test: `./bin/test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @elastic/react-search-ui-views@1.3.2 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jasonstoltzfus/.npm/_logs/2020-03-26T19_48_45_758Z-debug.log
```

Hopefully that is helpful